src/Matcher.php

```diff
@@ -60,6 +65,7 @@ public function startsWith($prefix): AbstractMatcher
     /**
      * {@inheritdoc}
      */
+    #[\Override]
```

Comment:
```suggestion
    #[Override]
```

`use Override;` しているので、 `\` は不要でしょうか？

Reply:
cs-fixの対象から漏れてましたね。対応しました！